microk8s-resources/actions/enable.falco.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+microk8s enable dns
+
+NAMESPACE_PTR="sysdig-agent"
+
+MANIFEST_PTR="https://raw.githubusercontent.com/draios/sysdig-cloud-scripts/master/agent_deploy/kubernetes/sysdig-agent-service.yaml"
+
+KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
+
+microk8s enable helm3
+
+# make sure the "sysdigagent" namespace exists
+$KUBECTL create namespace "$NAMESPACE_PTR" > /dev/null 2>&1 || true
+
+# load the CRD and wait for it to be installed
+$KUBECTL apply -f "$MANIFEST_PTR" -n "$NAMESPACE_PTR"
+
+echo "Enabling Auditlogging using webhook"
+mkdir "${SNAP_DATA}/args/auditlogging"
+cp "${SNAP}/actions/kube-api-audit" "${SNAP_DATA}/args/auditlogging"
+cp "${SNAP}/actions/webhook-config.yaml" "${SNAP_DATA}/args/auditlogging"
+echo "Reconfiguring apiserver"
+refresh_opt_in_config "audit-log-maxbackup" "3" kube-apiserver
+refresh_opt_in_config "audit-log-maxsize" "1024" kube-apiserver
+refresh_opt_in_config "audit-log-maxage" "30" kube-apiserver
+refresh_opt_in_config "audit-log-path" "/var/log/kube-apiserver-audit.log" kube-apiserver
+refresh_opt_in_config "audit-policy-file" "${SNAP_DATA}/args/auditlogging/kube-api-audit" kube-apiserver
+refresh_opt_in_config "audit-webhook-config-file" "${SNAP_DATA}/args/auditlogging/webhook-config-falco.yaml" kube-apiserver
+refresh_opt_in_config "audit-webhook-batch-max-wait" "5s" kube-apiserver
+AGENT_SERVICE_CLUSTERIP="$($KUBECTL get svc sysdig-agent -o=jsonpath={.spec.clusterIP} -n sysdig-agent)" envsubst < "${SNAP_DATA}/args/auditlogging/webhook-config.yaml" > "${SNAP_DATA}/args/auditlogging/webhook-config-falco.yaml"
+run_with_sudo preserve_env snapctl restart "${SNAP_NAME}.daemon-apiserver"
```

Comment:
This may not work on multi nodes.  It will only restart the apiserver on the node where the addon is enabled.
Need to use the helper function `restart_service apiserver`

Reply:
yes changed to helper function